### PR TITLE
Update the Makefile example

### PR DESCRIPTION
**Pull request by ChrisCummins**

### CODE REVIEW

**Comment by ChrisCummins** [comment]
Pushed an updated revision for +:100: :octocat: karma.

:+1: or :-1: ?

File: tools/Linux/makefile_basic/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Example Makefile to build a SYCL application using ComputeCpp.
+
+# Your ComputeCpp installation root.
+COMPUTECPP_PREFIX ?= /usr/local
+
+COMPUTECPP ?= $(COMPUTECPP_PREFIX)/bin/compute++
+COMPUTECPP_INFO ?= $(COMPUTECPP_PREFIX)/bin/computecpp_info
+COMPUTECPP_INCLUDES ?= $(COMPUTECPP_PREFIX)/include
+COMPUTECPP_LIBS ?= $(COMPUTECPP_PREFIX)/lib
```
Comment (ChrisCummins) [inline tools/Linux/makefile_basic/Makefile]: Dropped the intermediate `.o` file.

**Comment by DuncanMcBain** [comment]
Hi - this looks good to me, could you rebase and squash on master and I'll merge it up?

**Comment by ChrisCummins** [comment]
:ok_hand:

**Comment by DuncanMcBain** [comment]
Thanks very much!

**Pull request merged**